### ISSUE TITLE
Decimal type validation fails for scientific format

### ISSUE THREAD

**Issue by philiphendry** [dequelabs/axe-core#3531]
I'm running the Axe ruleset against our application using [Selenium.AXE for .NET](https://github.com/TroyWalshProf/SeleniumAxeDotnet). A third-party Blazor component we are using is generating aria-valuemin/max values and assigning the largest values for the .NET type bound to the component and in this case looks like this:

`<input role="spinbutton" aria-valuemin="-1.7976931348623157E+308" aria-valuemax="1.7976931348623157E+308" />`

We don't have control over these generated attributes and Axe reports this as a violation of the rule _aria-valid-attr-value_.

I'm very unfamiliar with your code base but tried to identify where the problem might reside and what I see is [some tests](https://github.com/dequelabs/axe-core/blob/95d37dd794dc8552d731fabf45244b260da53d8f/test/integration/rules/aria-valid-attr-value/aria-valid-attr-value.html#L233) that don't include scientific format and a regular [expression match for decimal](https://github.com/dequelabs/axe-core/blob/50492f772c4eb1565f6a2526d67779f89d800ccb/lib/commons/aria/validate-attr-value.js#L65) that also does not allow for scientific format.

**Comment by WilcoFiers** [comment]
Hey Philip. You may want to raise this with Blazor, rather than here. I'm going to go out on a limb here and guess those aren't actually the minimum and maximum values for that field. 

Running this in VoiceOver pretty quickly showed the problem:
![Voiceover announcing "NaN, Stepper"](https://user-images.githubusercontent.com/530687/177176434-752c22bd-4ec3-4644-9575-7ce5e94e3f2a.png)

This may valid number, but it's also definitely not something your screen reader users need to be informed of, and it breaks VoiceOver. This is a legit accessibility problem in my opinion. I don't think this needs a fix.

**Comment by philiphendry** [comment]
Thank you for your time looking at my issue @WilcoFiers. Given a screen reader can't read the value I can see it isn't reasonable to allow. I've gone down the road of raising an issue with the third-party component provider and see if there's a solution there.
